sources/web/datalab/polymer/components/datalab-files/datalab-files.ts

```diff
@@ -233,6 +234,8 @@ class FilesElement extends Polymer.Element {
         }
       })
       .then(() => this._fetching = false);
```

Comment:
Might want to clear _fileListFetchPromise here as well. If so, you might be able to remove _fetching and just use !!_fileListFetchPromise instead.